distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc

```diff
@@ -0,0 +1,76 @@
+:title: System Configuration Settings
+:type: installing
+:status: published
+:summary: Configure Single Sign On(SSO) page of installer.
+:project: ${branding}
+:order: 07
+
+.[[_configuring_sso]]Configure Single Sign On(SSO)
+****
+Configure Single Sign On method: SAML or OIDC.
+
+.SAML SSO
+Enter the URLs for the IdP metatdata and set other options.
+
+IdP Server::
+Configure ${BRANDING}'s' internal Identity Provider Server.
+
+. *_SP Metadata_*: The metatada of the desired Service Provider to connect to the internal IdP. Can be configured as an HTTPS URL (https://), file URL (file:), or an XML block(`&lt;md:EntityDescriptor&gt;...&lt;/md:EntityDescriptor&gt;`).
+
+. *_Require Signed AuthnRequests_*:
+Toggle whether or not to require signed AuthnRequests. When off, unsigned AuthnRequests will be rejected.
+
+. *_Limit RelayStates to 80 Bytes_*: Toggle whether or not to restrict the RelayState length to 80 bytes. When off, messages with RelayStates over 80 bytes will be rejected.
+
+. *_Cookie Expiration Time (minutes)_*:
+Sets the cookie expiration time for Single Sign On, which determines how long the internal IdP will cache SAML assertions for later use. This value should match the lifetime of SAML assertions.
+
+IdP Client::
+Configure handling of incoming requests where SAML authentication is enabled.
+
+. *_IdP Metadata_*:
+The metadata of the desired IdP to authenticate with. Can be configured as an HTTPS URL (https://), file URL (file:), or an XML block(&lt;md:EntityDescriptor&gt;...&lt;/md:EntityDescriptor&gt;).
```

Comment:
✏️ Same comment about the space after `block`.

Reply:
Also, should that block of code be inside a code fence? It is above.

Reply:
fixed